examples/web/config_root_charging_disabled.json

```diff
@@ -7,7 +7,7 @@
         "usb_charging_disabled_file": "sys/class/power_supply/battery/charging_enabled"
     }
    },
-  "replications": 1,
+  "repetitions": 1,
   "randomization": "False",
```

Comment:
Can you change also the examples, Eric, please? 
Here we still have the "string-version" of `randomization`.
I saw a similar bug also in other `config.json` files in the _examples_ directory.